### PR TITLE
the quality of image is also used for WebP

### PR DESCRIPTION
**Pull request by tudorbarascu**
the quality of the image parameter is also used in server WebP requests

### CODE REVIEW

File: src/ui/qgsprojectpropertiesbase.ui [hunk 1]
```diff
@@ -2370,7 +2370,7 @@
                           <item>
                            <widget class="QLabel" name="mWMSImageQualityLabel">
                             <property name="text">
-                             <string>Quality for JPEG images ( 10 : smaller image - 100 : best quality )</string>
+                             <string>Quality for JPEG ( 10 : smaller image - 100 : best quality ) and WebP images</string>
```
Comment (m-kuhn) [inline src/ui/qgsprojectpropertiesbase.ui:2373]: Also 10-100?

Comment (tudorbarascu) [reply src/ui/qgsprojectpropertiesbase.ui]: From 0 to 100, I redid the string to include this. Thanks!

Comment (m-kuhn) [reply src/ui/qgsprojectpropertiesbase.ui]: Thanks

1. there's a problem in formatting now (closing paranthesis for the first one), can you fix that?
2. the spacing around parantheses is different
3. it doesn't say for WebP which one is smaller in size/better quality, ideally we have the same level of information for all parts. If they are similar enough for JPEG and WebP, we can also just mention it once.

Comment (tudorbarascu) [reply src/ui/qgsprojectpropertiesbase.ui]: Done, sorry for the noise. Thank you.

File: src/ui/qgsprojectpropertiesbase.ui [hunk 2]
```diff
@@ -2370,7 +2370,7 @@
                           <item>
                            <widget class="QLabel" name="mWMSImageQualityLabel">
                             <property name="text">
-                             <string>Quality for JPEG images ( 10 : smaller image - 100 : best quality )</string>
+                             <string>Quality for JPEG and WebP images (from 0 to 100; 0 - lowest size, 100 - greater quality)</string>
```
Comment (m-kuhn) [inline src/ui/qgsprojectpropertiesbase.ui:2373]:
```suggestion
                             <string>Quality for JPEG and WebP images (from 0 to 100; 0 - smalles size, 100 - better quality)</string>
```

File: src/ui/qgsprojectpropertiesbase.ui [hunk 3]
```diff
@@ -2370,7 +2370,7 @@
                           <item>
                            <widget class="QLabel" name="mWMSImageQualityLabel">
                             <property name="text">
-                             <string>Quality for JPEG images ( 10 : smaller image - 100 : best quality )</string>
+                             <string>Quality for JPEG and WebP images (from 0 to 100; 0 - smalles size, 100 - better quality)</string>
```
Comment (m-kuhn) [inline src/ui/qgsprojectpropertiesbase.ui:2373]:
```suggestion
                             <string>Quality for JPEG and WebP images (from 0 to 100; 0 - smaller size, 100 - better quality)</string>
```

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
thanks @tudorbarascu

**Review by tudorbarascu — COMMENTED** [review]
smalles -> smallest

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**